sphinx/ext/autosectionlabel.py

```diff
@@ -20,7 +20,7 @@
 logger = logging.getLogger(__name__)
 
 
-def get_node_depth(node: Node) -> int:
+def _get_node_depth(node: Node) -> int:
```

Comment:
Ah maybe I wasn't clear. Old APIs are fine to be kept untouched (because we don't want to break packages) but it's good if *new* APIs can be designed with a good scope.